src/main/java/com/epam/postrest/client/PostUserClient.java

```diff
@@ -1,59 +0,0 @@
-package com.epam.postrest.client;
-
-import java.nio.charset.StandardCharsets;
-
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.http.HttpHeaders;
-import org.springframework.http.MediaType;
-import org.springframework.stereotype.Component;
-import org.springframework.util.LinkedMultiValueMap;
-import org.springframework.web.reactive.function.BodyInserters;
-import org.springframework.web.reactive.function.client.WebClient;
-
-import com.epam.postrest.domain.User;
-
-import reactor.core.publisher.Mono;
-
-@Component
-public class PostUserClient implements PostUser {
-
-    @Value("${baseUrl}")
```

Comment:
It's recommended to inject properties in constructor or in configuration classes. It's simple easier to test them.